package.json

```diff
@@ -1,5 +0,0 @@
-{
-  "dependencies": {
-    "webpack-bundle-analyzer": "^3.6.0"
```

Comment:
I think I saw contributions team using this the other day, but it feels like something we can install globally or in the project itself if we want it?

cc: @tjsilver @tomrf1

Reply:
I discovered that it's already installed in the parent package.json: https://github.com/guardian/support-frontend/blob/master/package.json

Reply:
we did briefly use it for comparing react vs preact. It's likely to be useful again, but can be addressed separately from this PR

Reply:
It is already in the package.json within `/support-frontend` I think it's just a mistake that it is here as well